compiler/ExtractBase.ml

```diff
@@ -1409,7 +1417,7 @@ let ctx_compute_type_name (span : Meta.span) (ctx : extraction_ctx)
     access then causes trouble because not all provers accept syntax like
     [x.3] where [x] is a tuple.
  *)
-let ctx_compute_field_name (span : Meta.span) (ctx : extraction_ctx)
+let ctx_compute_field_name (def : type_decl) (ctx : extraction_ctx)
     (def_name : llbc_name) (field_id : FieldId.id) (field_name : string option)
     : string =
   let field_name_s =
```

Comment:
You forgot to use the rename attribute provided for fields (below you're updating the prefix of the name we generate, in case we prefix the name of the field with the name of the structure).

Reply:
Why have you marked this conversation as revolved? The issue is still there...